odh/overlays/moc/kafka/topics/thoth-missing-package.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: cnv-prod.thoth.package-update.missing-package
```

Comment:
this one is the same as the one [here](https://github.com/operate-first/apps/pull/66/files#diff-6bd89ca4910acd4a1d011b1ec905c2eff1b1ed487b53cead6154710625b24149R4) -- can you pick a different topic name?

Reply:
oh thanks for the catch.
will fix this.

Reply:
the one in the thoth-missing-package-version is fixed.